section1/exercises/numbers.rb

```diff
@@ -4,13 +4,13 @@
 # `ruby section1/exercises/numbers.rb`
 
 # Example: Write code that prints the result of the sum of 2 and 2:
-p 2 + 2
+puts 2 + 2
 
 # Write code that prints the result of 7 subtracted from 83:
-p #YOUR CODE HERE
+puts 83 - 7
 
 # Write code that prints the result of 6 multiplied by 53:
-# YOUR CODE HERE
+puts 6 * 53
 
 # Write code that prints the result of the modulo of 10 into 54:
-# YOUR CODE HERE
+puts 10 % 54
```

Comment:
I think you read your instructions backwards. the modulo of 10 **into** 54 should be `54 % 10`